client/js/components/admin/instances/new_instance_modal.jsx

```diff
@@ -84,37 +85,37 @@ export default class NewInstanceModal extends React.Component {
             </div>
           </div>
           <div className="o-grid__item u-half">
-            <label className="c-input"><span>LTI Key</span><input type="text" /></label>
+            <label className="c-input" htmlFor="ltiKey"><span>LTI Key</span><input id="ltiKey" type="text" /></label>
```

Comment:
Complies with https://github.com/evcohen/eslint-plugin-jsx-a11y/blob/master/docs/rules/label-has-for.md